pyigm/abssys/dla.py

```diff
@@ -70,7 +72,7 @@ def from_datfile(cls, dat_file, tree=None, **kwargs):
 
         # QSO keys
         slf.qso = slf._datdict['QSO name']
-        slf.zqso = float(slf._datdict['QSO zem'])
+        slf.zem = float(slf._datdict['QSO zem'])
```

Comment:
Do we really have spaces in keywords? I guess no harm, but I don't like that too much :)